packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2727,6 +2727,7 @@
   "Old Colors": "Old Colors",
   "Old Colors (minor)": "Old Colors (minor)",
   "Older_than": "Older than",
+  "Omnichannel_Directory": "Omnichannel Directory",
```

Comment:
I didn't find any mention of this translation key, I think it's not longer used.